src/dysh/spectra/scan.py

```diff
@@ -1543,8 +1543,6 @@ class SubBeamNodScan(ScanMixin):
         Signal total power scans
     reftp:  list ~spectra.scan.TPScan
         Reference total power scans
-    fulltp:  ~spectra.scan.TPScan
-        A full (sig+ref) total power scans, used only for method='scan'
     method: str
```

Comment:
method argument is no longer needed/used, right?

Reply:
Correct. Removed.